aioxmpp/stream.py

```diff
@@ -2445,8 +2600,7 @@ def send(self, stanza, *, timeout=None):
                 )
             except asyncio.TimeoutError:
                 raise TimeoutError
-
```

Comment:
Not sure why this changed; since the tests pass it probably is correct.

Reply:
(This refers to the change on the return argument of send below not shown in the context lines).

Reply:
See the ``handler_ok``, it already unwraps the payload from the stanza.

We had to do this because ``cb`` can return anything, not only the stanza.